cmd/sommelier/cmd/gentx.go

```diff
@@ -0,0 +1,539 @@
+package cmd
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+	"sort"
+	"strings"
+
+	"github.com/pkg/errors"
```

Comment:
nit: use `fmt.Errorf` to be consistent with the rest of the repo

Reply:
kk